supertokens_python/querier.py

```diff
@@ -285,10 +285,17 @@ async def __send_request_helper(
                     + response.text  # type: ignore
                 )
 
+            res: Dict[str, Any] = {"_headers": dict(response.headers)}
+
+            if response.headers.get("content-type", "").startswith("text"):
+                res["_text"] = response.text
+
             try:
-                return response.json()
+                res.update(response.json())
             except JSONDecodeError:
-                return response.text
+                pass
```

Comment:
this is changing the semantics of how things were done. Please fix

Reply:
Done